portsyncd/portsyncd.cpp

```diff
@@ -154,12 +154,17 @@ void handlePortConfigFile(ProducerTable &p, string file)
         }
 
         istringstream iss(line);
-        string alias, lanes;
-        iss >> alias >> lanes;
+        string name, lanes, alias;
+        iss >> name >> lanes >> alias;
```

Comment:
I believe the current implementation is okay since if the alias is specifically specified, then we will have it, otherwise we don't.